targets/posix/nanoCLR/posix_stubs.cpp

```
@@ -0,0 +1,144 @@
+extern const CLR_RT_NativeAssemblyData g_CLR_AssemblyNative_System_Math;
+extern const CLR_RT_NativeAssemblyData g_CLR_AssemblyNative_System_Runtime_Serialization;
+extern const CLR_RT_NativeAssemblyData g_CLR_AssemblyNative_nanoFramework_ResourceManager;
+extern const CLR_RT_NativeAssemblyData g_CLR_AssemblyNative_nanoFramework_System_IO_Hashing;
+extern const CLR_RT_NativeAssemblyData g_CLR_AssemblyNative_nanoFramework_Runtime_Events;
+extern const CLR_RT_NativeAssemblyData g_CLR_AssemblyNative_nanoFramework_Runtime_Events_EventSink_DriverProcs;
+extern const CLR_RT_NativeAssemblyData g_CLR_AssemblyNative_System_Device_Gpio;
+extern const CLR_RT_NativeAssemblyData g_CLR_AssemblyNative_System_Device_I2c;
+extern const CLR_RT_NativeAssemblyData g_CLR_AssemblyNative_System_Device_Spi;
+extern const CLR_RT_NativeAssemblyData g_CLR_AssemblyNative_System_IO_Ports;
+
+const CLR_RT_NativeAssemblyData *g_CLR_InteropAssembliesNativeData[] = {
```

Clang (14.0.6)

[warning] variable 'g_CLR_InteropAssembliesNativeData' is non-const and globally accessible, consider making it const

(cppcoreguidelines-avoid-non-const-global-variables)